contract_variable_quantity/models/contract_template_line.py

```diff
@@ -6,8 +6,8 @@
 from odoo import fields, models
 
 
-class ContractAbstractContractLine(models.AbstractModel):
-    _inherit = "contract.abstract.contract.line"
+class ContractAbstractContractLine(models.Model):
+    _inherit = "contract.template.line"
```

Comment:
Maybe rename the python class, as it makes no sense anymore to keep "Abstract" inside its name?

Reply:
ok